ajax/libs/gumshoe/package.json

```diff
@@ -7,6 +7,12 @@
     "name": "Chris Ferdinandi",
     "url": "http://gomakethings.com"
   },
+  "keywords": [
+    "gumshoe",
+    "html",
+    "javascript",
+    "gulp"
```

Comment:
I guessed it is the important tool related with `gumshoe`.
But now I think it is just dependency about `gumshoe`.
So maybe I should remove it, right?
Thanks!